activerecord/lib/active_record/associations.rb

```diff
@@ -1350,6 +1352,7 @@ module ClassMethods
         # [:inverse_of]
         #   Specifies the name of the #belongs_to association on the associated object
         #   that is the inverse of this #has_many association.
+        #   Note that it will be automatically detected if the association matches the name of the class.
```

Comment:
As mentioned before, there's a section earlier on in this file that's dedicated to inversing associations. I like this duplication if you're just reading the docs on a specific method but its probably better to just point to that inversing section instead.